pkg/cmd/server/api/types.go

```diff
@@ -8,8 +8,8 @@ import (
 
 const (
 	FeatureBuilder    = `Builder`
-	FeatureS2I        = `S2I Builder`
-	FeatureWebConsole = `Web Console`
+	FeatureS2I        = `S2IBuilder`
```

Comment:
add a comment to this set of constants that any changes require the addition of an alias for backwards compatibility